### PR TITLE
feat: docs ai warning removal

### LINKED ISSUE #22714: Ask AI getting unusable.  Why popup so small and hide output?
# Improve documentation

## Describe the problem

Supabase AI is great in helping me work and reson through what I need to accomplish but I get only a small view of response, requiring a lot of scroll and the worst part is the block "Supabase AI is experimental and may produce incorrect answers. Always verify the output before executing." sits on top of the response text requiring me to copy/paste the last part of the results in anpther doc to see what it says.  It interrupts workflow - once or twice not bad but after doing this a dozen times gets annoying.

You got a great AI feature.  I do not understand the reason why you would make it so inaccessible.

## Describe the improvement

Make the ASK AI popup full screen,  remove the "Supabase AI is experimental and may produce incorrect answers." or at least allow my to close it or scroll it out or hide so I can see the full output.

Always verify the output before executing.
## Additional context

image

### PR DESCRIPTION
**Pull request by Hallidayo**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Feature - Supabase Docs - Ask AI

## What is the current behavior?

Their is an alert which appears which tells the user about the ai being experimental. This makes the page of the window really small.

## What is the new behavior?

Experimental button is now a tooltip and the warning is removed:

https://github.com/supabase/supabase/assets/22655069/4e61c8bd-8af9-44ef-ab6c-d7323c92c32b

## Additional context

Closes #22714

### CODE REVIEW

**Review by charislam — CHANGES_REQUESTED** [review]
Thanks @Hallidayo! I'd rather keep a very explicit, in-your-face warning message about this. I think it's important to prompt users to take a pause for reflection before running anything against their database, and the tooltip is easy to ignore.

There's probably room to expand the height of the entire dialog on most screens so you have more reading space despite the disclaimer, perhaps we can do that instead?

**Comment by Hallidayo** [comment]
Hi @charislam - Just reverted the original changes and increased the window a little as well:

Screenshot 2024-05-01 at 22 05 51

File: packages/ui-patterns/Cmdk/CommandMenu.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const CommandMenu = () => {
     currentPage === COMMAND_ROUTES.AI ||
     currentPage === COMMAND_ROUTES.GENERATE_SQL
       ? 'min(600px, 50vh)'
```
Comment (charislam) [inline packages/ui-patterns/Cmdk/CommandMenu.tsx:87]: Thanks @Hallidayo! Really appreciate you going along with the changes 😄 IIRC it is this line actually that sets the height on the AI pages (it's already bigger than the usual but I guess we can increase it even further)

**Comment by Hallidayo** [comment]
Hi @charislam - Thanks for having a look at this, I've just increase the content height a little as well

**Comment by charislam** [comment]
Thanks @Hallidayo and sorry for the delay in getting back to this! Height looks good, but the backdrop filter seems to have disappeared (might be fixed by merging and fixing conflicts 🤔 perhaps)

image

**Comment by Hallidayo** [comment]
Hi @charislam -  no problem thank you for looking over this one! Just resolved the merge conflict now.

**Review by charislam — APPROVED** [review]
Awesome, thanks!

Checks: checked command menu in preview (docs)

**Pull request merged**